src/components/ThemeProvider/ThemeProvider.tsx

```diff
@@ -55,20 +62,26 @@ export class ThemeProvider extends React.Component<ThemeProviderProps, State> {
 
     return (
       <ThemeContext.Provider value={theme}>
-        <div style={styles}>{children}</div>
+        <div style={legacyStyles}>
```

Comment:
Adding an extra div with legacy `topBar` styles feels like a nice separation of concerns vs. cramming these all on one div.

Reply:
Wrapper divs have caused us some pain before - in UXPin integration and people raising an eyebrow as styleguide's html output includes them. I appreciate it's a nice separation of concerns but I'd like to avoid adding extra nesting if possible. If the only reason for separating them is nice aesthetics could we keep to a single div to reduce the chance people thinking multiple wrapping divs is odd.